src/page/OrderPage.tsx

```diff
@@ -1,76 +1,118 @@
-import { useLocation, useNavigate } from 'react-router';
+import { useNavigate } from 'react-router';
+import * as styles from './OrderPage.styles';
 import Header from '../components/common/Header';
 import Button from '../components/common/Button';
-import { css } from '@emotion/react';
-import { useEffect } from 'react';
+import { useState } from 'react';
+import CouponModal from '../components/Modal/CouponModal';
+import CheckBox from '../components/common/CheckBox';
+import { getLocalStorage, setLocalStorage } from '../utils/localStorage';
+import PriceRow from '../components/PriceArea/PriceRow';
+import * as Card from '../components/Card/Card';
+import { calculateDeliveryFee } from '../utils/coupon/calculate';
+
+export type SelectedItem = {
+  id: number;
+  name: string;
+  quantity: number;
+  price: number;
+  imageUrl: string;
+};
 
 function OrderPage() {
+  const [isCouponModalOpen, setIsCouponModalOpen] = useState(false);
+  const [isRemoteArea, setIsRemoteArea] = useState(() => localStorage.getItem('isRemoteArea') === 'true');
+  const [totalCouponDiscount, setTotalCouponDiscount] = useState(0);
   const navigate = useNavigate();
-  const location = useLocation();
-  const { countOfItem, countOfItemType, totalAmount } = location.state ?? {};
+  const items = getLocalStorage<SelectedItem[]>('selectedItems', []);
+
+  const totalQuantity = items.reduce((acc, item) => acc + item.quantity, 0);
+  const totalOrderAmount = items.reduce((acc, item) => acc + item.price * item.quantity, 0);
+
+  const basicDeliveryFee = calculateDeliveryFee(totalOrderAmount);
+  const remoteAreaExtraFee = isRemoteArea ? 3000 : 0;
```

Comment:
소소하게 numeric seprator를 적용하셔도 좋을거 같아요
```suggestion
  const remoteAreaExtraFee = isRemoteArea ? 3_000 : 0;
```